pkg/deploy/rp.go

```diff
@@ -48,7 +48,53 @@ func newGenerator(production bool) *generator {
 	}
 }
 
-func (g *generator) vnet() *arm.Resource {
+func (g *generator) peering(vnetA string, vnetB string) []*arm.Resource {
```

Comment:
think this function could be half the size and have the caller call it twice:
```
peering(a, b)
peering(b, a)
```

Reply:
Changed to halfPeering(a,b);halfPeering(b,a); Named to make the necessity for both (a,b) and (b,a) obvious to anyone reading the code (azure portal does both when configuring a peering from one side).